src/delta.rs

```diff
@@ -466,11 +463,10 @@ fn handle_hunk_header_line(
                 config.hunk_header_style,
                 decoration_ansi_term_style,
             )?;
-            if !config.hunk_header_style.is_raw {
-                painter.output_buffer.clear()
-            };
+            painter.output_buffer.clear();
```

Comment:
What bug is this fixing?

Reply:
Would you be able to add test coverage for the changes in this file? Specifically, when I revert the changes in this file like this:

```
git checkout master src/delta/src/delta.rs
```

right now, the full test suite still passes. I'd really like to have tests that fail when the changes in this file are reverted. Do you think you have time do do that?

Reply:
I should have left comment.
This if statement is not needed because when config.hunk_header_style.is_raw, it won't come to this process.

https://github.com/ryuta69/delta/blob/4fd75f1b9f4abd1040126f2e87bdf4508947e200/src/delta.rs#L420

Reply:
@dandavison

So, this is not a bug, but i just refactored (again, I should separate PR. I'm sorry...).
The tests were not failed before this fix.

Reply:
I see, thanks!

OK, so one of us should add test coverage for the #293 fix. Do you feel like doing that?

Reply:
Yes!
As I stated `I will add test for this` in Remark, I am going to make a test for this!

Reply:
Oh yes :) Awesome, I'll let you do it in a separate PR.

Reply:
Ok!